src/Compilers/Core/Portable/Symbols/ITypeSymbol.cs

```diff
@@ -168,6 +168,17 @@ public interface ITypeSymbol : INamespaceOrTypeSymbol
             NullableFlowState topLevelNullability,
             int position,
             SymbolDisplayFormat? format = null);
+
+        /// <summary>
+        /// Nullable annotation associated with the type, or <see cref="NullableAnnotation.None"/> array if there are none.
```

Comment:
>array [](start = 99, length = 5)

array?

Reply:
I can't see any delta in which this is addressed (in CodeFlow)

---
In reply to: [338834332](https://github.com/dotnet/roslyn/pull/39498#discussion_r338834332) [](ancestors = 338834332)

Reply:
> I can't see any delta in which this is addressed (in CodeFlow)

It is addressed in the 5th commit. It is known that CodeFlow is unable to show all the combined changes.

---
In reply to: [339782147](https://github.com/dotnet/roslyn/pull/39498#discussion_r339782147) [](ancestors = 339782147,338834332)